src/iexamine.cpp

```diff
@@ -4040,7 +4042,7 @@ void iexamine::autodoc( player &p, const tripoint &examp )
                 comps.push_back( item_comp( it->typeId(), 1 ) );
                 p.consume_items( comps, 1, is_crafting_component );
                 if( needs_anesthesia ) {
-                    p.consume_items( acomps, 1, is_crafting_component );
+                    anesth_kit[0]->ammo_consume( 1, p.pos() );
```

Comment:
There's no reason not to use a `tool_comp` and `player::consume_tools()` here.

Reply:
Done